src/main/js/posts/inputPost.js

```diff
@@ -14,7 +14,7 @@ class InputPost extends React.Component {
                 </h1>
                 <form>
                     <textarea name="message" placeholder="Don't worry, be happy"/>
-                    <button type="submit" name="Take a Longshot" value="Take a Longshot">Take a Longshot</button>
+                    <button type="submit" name="submitPost">Take a Longshot</button>
```

Comment:
As per @AlexDresco and @shwetzpatil comment I've removed the value, however I left the name so we have something to refer to the button by when testing, or when calling an onClick event